### DISCUSSION TITLE
Porter installations

### DISCUSSION THREAD

**Discussion by dejanu** [getporter/porter#3184] [Q&A] [answered]
### Why does Porter allow the following combinations?

I'm using `porter v1.1.0 (b50c1899)` and I was wondering why, all these flag combinations are available.
 ```bash
porter list
porter installation list
porter installations list
```
Is there any difference between them? At first glance, it does not seem. All 3 show bundle metadata. Thx

**Comment by kichristensen** [comment, marked as answer]
@dejanu they are now difference between them, `porter list` and `porter installation list` are aliases for `porter installations list`.
The aliases are created for convenience only.
I hope it answers you question, otherwise please let me know

**Reply by dejanu** [reply]
I see, just discovered  `aliases.go`. @kichristensen  Thx
